godtossing/Pos/Ssc/GodTossing/Arbitrary.hs

```diff
@@ -128,6 +136,9 @@ instance Arbitrary Opening where
 
 instance Arbitrary VssCertificate where
     arbitrary = mkVssCertificate <$> arbitrary <*> arbitrary <*> arbitrary
+    shrink VssCertificate {..} = [ VssCertificate { vcExpiryEpoch = shrunkEpoch, .. }
+                                 | shrunkEpoch <- shrink vcExpiryEpoch
```

Comment:
It seems broken to me. If you change `vcExpiryEpoch` in `VssCertificate`, the signature inside it becomes invalid.